src/pg_type_template/templates/test/expected/type_name_004_parallel.out.jinja

```diff
@@ -1,8 +1,15 @@
 \t
 SET max_parallel_workers_per_gather=4;
-SET force_parallel_mode=on;
-set parallel_setup_cost = 10;
-set parallel_tuple_cost = 0.001;
+SET parallel_setup_cost = 10;
+SET parallel_tuple_cost = 0.001;
+DO $$
+BEGIN
+    IF current_setting('server_version_num')::int >= 160000 THEN
+        EXECUTE 'SET debug_parallel_query = on';
```

Comment:
Thank you @ridvankorkmaz for the PR.
Is I may ask, why do you use `EXECUTE` in this case?
Just using `SET` directly would have the same effect wouldn't it?

Reply:
Thank you @mrimbault for review,
I tested it, yes both have same effect, so both are OK.
I would like to say I prefer to go without `EXECUTE` but, in this case we may need to modify all following repositories. IMHO, does not worth that.
What do you prefer?

https://github.com/adjust/active_events/pull/11
https://github.com/adjust/pg_parquet_discovery/pull/47
https://github.com/adjust/pg-purchase_verification_status/pull/1
https://github.com/adjust/pg-adkit_types/pull/4
https://github.com/adjust/pg-match_type/pull/6
https://github.com/adjust/pg-rejected_engagement_type/pull/6
https://github.com/adjust/pg-sk_coarse_conversion_value/pull/4
https://github.com/adjust/pg-sk_condition_type/pull/7
https://github.com/adjust/pg-subscription_event_type/pull/13
https://github.com/adjust/pg-unverified_install_reason/pull/2

Reply:
Thank you @ridvankorkmaz.

I also double checked [the doc](https://www.postgresql.org/docs/current/plpgsql-statements.html#PLPGSQL-STATEMENTS-EXECUTING-DYN), just in case.
Here are the only differences between using `EXECUTE` or not for a query that is not dynamically generated:

> An EXECUTE with a simple constant command string and some USING parameters, as in the first example above, is functionally equivalent to just writing the command directly in PL/pgSQL and allowing replacement of PL/pgSQL variables to happen automatically. The important difference is that EXECUTE will re-plan the command on each execution, generating a plan that is specific to the current parameter values; whereas PL/pgSQL may otherwise create a generic plan and cache it for re-use. In situations where the best plan depends strongly on the parameter values, it can be helpful to use EXECUTE to positively ensure that a generic plan is not selected.

So, in this case it probably wouldn't make any significant difference in the execution.

To me the problem is one of code clarity and meaningfulness.
Using `EXECUTE` in this context has no meaning, no purpose, it is just noise and may cause confusion.

So my personal preference would be to fix it, and use the proper syntax - including in all the other affected repositories.

Reply:
For the sake of prioritization, I will without change and take a note for future.